clients/src/main/java/org/apache/kafka/common/config/LogLevelConfig.java

```diff
@@ -0,0 +1,60 @@
+package org.apache.kafka.common.config;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * This class holds definitions for log level configurations related to Kafka's application logging. See KIP-412 for additional information
+ */
+public class LogLevelConfig {
+    /*
+     * NOTE: DO NOT CHANGE EITHER CONFIG NAMES AS THESE ARE PART OF THE PUBLIC API AND CHANGE WILL BREAK USER CODE.
+     */
+
+    /**
+     * The <code>FATAL</code> level designates a very severe error
+     * that will lead the Kafka broker to abort.
+     */
+    public static final String FATAL_LOG_LEVEL = "FATAL";
+
+    /**
+     * The <code>ERROR</code> level designates error events that
+     * might still allow the broker to continue running.
+     */
+    public static final String ERROR_LOG_LEVEL = "ERROR";
+
+    /**
+     * The <code>WARN</code> level designates potentially harmful situations.
+     */
+    public static final String WARN_LOG_LEVEL = "WARN";
+
+    /**
+     * The <code>INFO</code> level designates informational messages
+     *      that highlight normal Kafka events at a coarse-grained level
+     */
+    public static final String INFO_LOG_LEVEL = "INFO";
+
+    /**
+     * The <code>DEBUG</code> Level designates fine-grained
+     * informational events that are most useful to debug Kafka
+     */
+    public static final String DEBUG_LOG_LEVEL = "DEBUG";
+
+    /**
+     * The <code>TRACE</code> Level designates finer-grained
+     * informational events than the <code>DEBUG</code level.
+     */
+    public static final String TRACE_LOG_LEVEL = "TRACE";
+
+    /**
+     * The <code>ALL</code> has the lowest possible rank and is intended to turn on all logging.
+     */
+    public static final String ALL_LOG_LEVEL = "ALL";
```

Comment:
In reality, `TRACE` does the same job. I don't have a strong opinion whether we should support this or not.

Reply:
Agreed. I did not know about this log level until I dove into the log4j java docs